index.css

```diff
@@ -287,39 +333,77 @@ h4 {
         justify-content: space-around;
         order: 1;
     }
-}
 
-/* iframe {
-    width: 100%;
-    height: 77px;
-    border:0;
-    transform: scale(1);
-    overflow:hidden;
-} */
-
-.doc-container {
-    position: relative;
-    display: flex;
-    flex-direction: row;
-    justify-content: flex-start;
-    margin: 2rem 12rem;
-}
+    .doc-container {
+        position: relative;
+        display: flex;
+        flex-direction: row;
+        justify-content: flex-start;
+        margin: 2rem 8rem;
```

Comment:
maybe u could make a utility class out of display: flex , flex-direction : row , justify-content-flex-start as u r using these repeatedly
PS: is flex-direction: row required here? since row is the default value for flex-direction